src/Raven.Server/Documents/Sharding/Commands/GetDocumentConflictsCommand.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Raven.Client.Documents.Commands;
+using Raven.Client.Http;
+using Raven.Server.Documents.Sharding.Handlers;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.Sharding.Commands
+{
+    internal class GetDocumentConflictsCommand : ShardedStreamCommand
```

Comment:
let's kill this and simply use the `GetReplicationConflictsCommand` instead

Reply:
done

Reply:
?

Reply:
done